devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -128,31 +150,60 @@ def parse(String description) {
 	}
 }
 
+def off() {
+	zigbee.off() +
+    sendEvent(name: "level", value: 0)
```

Comment:
Are these events necessary? Typically we let the device report its new state and then generate the appropriate events.

Reply:
This is necessary for my device as it doesn't send a level event when it is told to fully open or close.